### DISCUSSION TITLE
are there kirkstone branches for l4t 35.3.x, 35.4.x ?

### DISCUSSION THREAD

**Discussion by noob502** [OE4T/meta-tegra#1620] [Q&A]
Hello, 
I am looking for meta-tegra Kirkstone for 35.3.x and 35.4.x. I found the commit, but they have already been merged. Is there any way? 
Thank you.

**Comment by madisongh** [comment]
You can maintain your own - fork and pull a branch off the last commit for the 35.x version you're targeting, then look for later patches not related to the BSP upgrade that you should cherry-pick to your branch.
